### ISSUE TITLE
Inconsistent UIA IsOffscreen and ClickablePoint properties for WPF ListViewItems that are scrolled out of view

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4631]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`) 3.0+
* Windows version: (`winver`) all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
Copied from internal .NET Framework bug [1161493](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1161493), which contains additional information.

 **Problem description:**
1. Create a new WPF .NET Framework 4.8 app.
2. Add the following XAML:

        <ListView Height="60">
            <ListViewItem Content="One" />
            <ListViewItem Content="Two" />
            <ListViewItem Content="Three" />
            <ListViewItem Content="Four" />
            <ListViewItem Content="Five" />
            <ListViewItem Content="Six" />
            <ListViewItem Content="Seven" />
            <ListViewItem Content="Eight" />
            <ListViewItem Content="Nine" />
            <ListViewItem Content="Ten" />
        </ListView>

3. Run the app, verify that the ListView can only present a subset of the full item set, and scroll through the list such that all items have been viewed.

4. Point the Accessibility Insights for Windows (AIWin) tool at the list, and review the failures that the tool reports.

Result: AIWin reports a failure of "An element's IsOffScreen property must be false when its clickable point is on screen" for all items scrolled out of view. 

Expected: AIWin doesn't report failures for the default WPF experience. 

This also happens for TreeView and DataGrid.

**Comment by SamBent** [comment]
Fixed in 6.0-rc1.
